packages/manager/apps/dedicated/client/app/dedicated/server/monitoring/monitoring.html

```diff
@@ -464,7 +464,7 @@ <h2 data-translate="server_tab_MONITORING_services"></h2>
         >
             <!-- monitoring services -->
             <tr
-                data-ng-repeat="monitoring in $ctrl.monitoringServices track by monitoring.monitoringId"
+                data-ng-repeat="monitoring in $ctrl.monitoringServices track by $index"
```

Comment:
`monitoring.monitoringId` may not be unique ?

Reply:
no not when there are 2 monitoring entries for same IP